hw5-10_to_do_list/react_ts/src/api/WeatherAPI.ts

```diff
@@ -0,0 +1,22 @@
+export default class WeatherAPI {
```

Comment:
```suggestion
export class WeatherAPI {
```
I don't recommend to use default exports